doc/source/toolkit/index.rst

```diff
@@ -5,26 +5,19 @@
 API reference
 =============
 
-This section provides descriptions of the three APIs available for the Antenna Toolkit:
+This section provides descriptions of the two APIs available for the Antenna toolkit:
 
 - **Toolkit API**: Contains the ``Toolkit`` class, which provides methods for
-  controlling the toolkit workflow. In addition to methods for creating an AEDT
-  session or connecting to an existing AEDT session, this API provides methods
+  controlling the toolkit workflow. This API provides methods
   for synthesizing and creating an antenna. You use the Toolkit API at the
   toolkit level.
 
 - **Antenna API**: Contains classes for all antenna types available in the toolkit.
```

Comment:
Here, what toolkit is meant in "available in the toolkit"? You can see how not using the full "product name" could lead to confusion. I "think" you are referring to the AEDT Antenna Toolkit" here.